sphinx/util/docfields.py

```diff
@@ -54,35 +54,36 @@ def __init__(self, name, names=(), label=None, has_arg=True, rolename=None,
         self.rolename = rolename
         self.bodyrolename = bodyrolename
 
-    def make_xref(self, rolename, domain, target,
+    def make_xref(self, env, rolename, domain, target,
```

Comment:
I think this change might be incompatible changes.
We provides domains feature as an extensible point of Sphinx.
So I'm worry about any 3rd party domain uses these functions in docfields.

How about adding this as a optional argument?
It does not breaks the interface.